test/browser/render.js

```diff
@@ -588,44 +699,69 @@ describe('render()', () => {
 		expect(sortAttributes(html)).to.equal(sortAttributes('<input type="range" min="0" max="100" list="steplist">'));
 	});
 
-	it('should not execute append operation when child is at last', (done) => {
+	it('should not execute append operation when child is at last', () => {
+		// See developit/preact#717 for discussion about the issue this addresses
```

Comment:
👍 I wish I'd left more of these issue references around.

Reply:
side note: we should totally look into reusing this component + plumbing for a bunch of different tests.